docs/guides/dkms.md

```diff
@@ -134,6 +134,36 @@ It unloads the Touchbar modules as they can cause issues for suspend.
 
 Your keyboard backlight may remain switched off on resuming and backlight controls may stop working. A restart fixes the backlight controls. You may also run `echo 60 > /sys/class/leds/apple::kbd_backlight/brightness` to turn on the backlight to the maximum level if you do not want to boot. Replace 60 with a lower number for lower brightness.
```

Comment:
I don't think this part is needed now.

Reply:
I went trying out installing the package again inside a distrobox and it seems like that when installing it on Arch Linux doesn’t seem to create this file, however it does when installing it from the GitHub repository.

Reply:
Which file do you mean? the udev rule or /sys/class/leds/apple::kbd_backlight/brightness maybe? Also which package did you install?

Reply:
The udev rule. Installing the "acpilight" package from pacman doesn't seem to setup the udev rule file, unlike installing it from the git.

Reply:
Did you check in /etc? pacman -Ql acpilight will list what files the package contains, and I think it would have put the rule in /usr/lib